### ISSUE TITLE
Targets older Opera.exe

### ISSUE THREAD

**Issue by Dacke** [karma-runner/karma-opera-launcher#6]
Currently the launcher looks for the opera.exe file instead of the newer launcher.exe.  Since the new version of the browsers are based on the Chromium open source project.  They do not use the older classic implementation.
Can the launcher be changed to either work with both or just updated to use the new one?

**Comment by Dacke** [comment]
Modified the index.js, OperaBrowser.prototype to the following:
````javascript
OperaBrowser.prototype = {
  name: 'Opera',

  DEFAULT_CMD: {
    linux: 'opera',
    darwin: '/Applications/Opera.app/Contents/MacOS/Opera',
    win32: process.env.ProgramFiles + ' (x86)\\Opera\\Launcher.exe'
  },
  ENV_CMD: 'OPERA_BIN'
};
````
It opens 6 tabs that don't work out and then produces this output in the console window.
*INFO [karma]: Karma v0.12.31 server started at http://localhost:9876/
INFO [launcher]: Starting browser Opera
ERROR [launcher]: Cannot start Opera

INFO [launcher]: Trying to start Opera again (1/2).
ERROR [launcher]: Cannot start Opera

INFO [launcher]: Trying to start Opera again (2/2).
ERROR [launcher]: Cannot start Opera

ERROR [launcher]: Opera failed 2 times (cannot start). Giving up.*

**Comment by Dignifiedquire** [comment]
Should be fixed by #12
